docs/overview.rst

```diff
@@ -0,0 +1,82 @@
+.. _overview:
+
+:github_url: https://github.com/jrbourbeau/pyunfold
+
+************
+Overview
+************
+
+PyUnfold is a Python language software package for the incorporation of the imperfections
+of the measurement process in the data analysis roadmap.
+
+In an ideal world, scientists would have access to the perfect detector:
+an apparatus that makes no error in measuring a desired quantity.
+Real detectors have finite resolutions, less than 100% efficiencies, and can have 
+characteristic biases that may not be eliminable, yet themselves can be measured or estimated.
+Statistical uncertainties also propagate through the measurement process, inclusion
+of which typically requires either careful book-keeping or conservative estimates.
+
+By building a matrix that encodes a detector's smearing of the desired true quantity
+into the measured observable(s), a deconvolution can be performed that backs out
+an estimate of the true variable.
+The unfolding method implemented in PyUnfold accomplishes this deconvolution
+by harnessing the power of Bayes' Theorem in an iterative procedure, providing results
+based on physical expectations of the desired quantity.
+Furthermore, tedious book-keeping for both statistical and systematic errors
+produces precise final uncertainty estimates.
+
+
+-----------
+Terminology
+-----------
+
+In the measurement process, a true distribution of **causes** is detected by some
+apparatus having a characteristic **response function** or matrix which produces a
+*measurable* distribution of **effects**.
+The act unfolding approximates the inverse of the response function by convolving it with
+a **prior** distribution which encodes a guess as to the form of the true cause distribution.
+Including the measured effects distribution, we obtain an updated, better guess of the cause
+distribution.
+This update can be used as a prior for another unfolding, making this an iterative procedure.
+The iterations stop once a test statistic comparing the unfolded cause distribution between 
+two iterations satisties some pre-defined threshold, typically taking just a few iterations.
+One needs only to provide the response matrix, its estimated uncertainties, and the measured 
+effects distribution to use PyUnfold.
+
+
+
+------------------
+Who uses PyUnfold?
+------------------
+
+The potential audience for PyUnfold includes natural scientists who are in need of a 
+statistical and programming framework in which to incorporate the uncertainties of 
+their measurement processes to estimate their desired variables.
+PyUnfold already has been successfully implemented to unfold the cosmic-ray energy 
+spectrum using data from the HAWC_ and IceCube_ observatories.
```

Comment:
The links here are showing up with the trailing underscore in the rendered documentation page. Something like 
```
`HAWC <https://www.hawc-observatory.org/>`_
```

should work